tsconfig.json

```diff
@@ -18,6 +18,7 @@
       "app.constants": ["core/templates/dev/head/app.constants"],
       "components/*": ["core/templates/dev/head/components/*"],
       "domain/*": ["core/templates/dev/head/domain/*"],
+      "expressions/*": ["core/templates/dev/head/expressions/*"],
```

Comment:
Why is this change required?

Reply:
I believe the code does not compile without it because it cannot find the `expression-parser.service.ts` file in that directory. Prior to my changes, `ExpressionParserService` was a JavaScript file that did not need compilation, but since I have upgraded it to a TypeScript module, it must be compiled.

Here is the output I get when I run the frontend tests without that line in `tsconfig.json`:
![image](https://user-images.githubusercontent.com/6963537/73864951-7c482180-4810-11ea-90d8-4a893b822361.png)

Reply:
The paths here are added to resolve path names while importing i.e. `expressions/[abc]` maps to `core/templates/dev/head/expressions/[abc]`. From the screenshot, the error is happening due to this line https://github.com/oppia/oppia/pull/8557/files#diff-eba3efa3e78f5b0ab9ed5207732639b4R23. Somehow, parser cannot be imported which results in an error in expression-parser.service and it cannot be imported as well. Can you please find the cause of why this error is happening rather than removing the line here since we should have a clear idea of why the module is not found for this case though it is working fine for all the other folders before removing this line.

Reply:
Could you explain a bit more? I am not removing a line, but rather adding a line in order to allow TypeScript to find the file.

Reply:
Ok, my bad. I think I did not see this change correctly. Sorry for the misunderstanding. It's strange that this line was missing before. Thanks for adding!